build/source/driver/multi_driver.f90

```diff
@@ -574,16 +602,35 @@ program multi_driver
 
 end do  ! (looping through GRUs)
 
-! initialize time step index
-jstep=1
+! *****************************************************************************
+! (5e) initialize first output sequence 
+! *****************************************************************************
+! define the file
+! NOTE: currently assumes that nSoil is constant across the model domain
+write(fileout,'(a,i0,a,i0,a)') trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//'_spinup'//trim(output_fileSuffix)
+call def_output(nHRU,gru_struc(1)%hruInfo(1)%nSoil,fileout,err,message); call handle_err(err,message)
+ 
+! write local model attributes and parameters to the model output file
+do iGRU=1,nGRU
+ do iHRU=1,gru_struc(iGRU)%hruCount
+  call writeParm(iHRU,attrStruct%gru(iGRU)%hru(iHRU)%var,attr_meta,err,message); call handle_err(err,message)
+  call writeParm(iHRU,typeStruct%gru(iGRU)%hru(iHRU)%var,type_meta,err,message); call handle_err(err,message)
+  call writeParm(iHRU,mparStruct%gru(iGRU)%hru(iHRU)%var,mpar_meta,err,message); call handle_err(err,message)
+ enddo ! HRU
+ call writeParm(integerMissing,bparStruct%gru(iGRU)%var,bpar_meta,err,message); call handle_err(err,message)
+enddo ! GRU
 
 ! ****************************************************************************
 ! (6) loop through time
 ! ****************************************************************************
-do istep=1,numtim
+! initialize time step index
+waterYearTimeStep = 1
+do iFreq = 1,nFreq; outputTimeStep(iFreq) = 1; enddo
```

Comment:
I agree -- this was likely a result of me being unclear in code ideas. In initial code walk through I meant `outputTimeStep(:) = 1` instead of `outputTimeStep=1` to make it clear that you are initializing a vector. Michael's suggestion of `outputTimeStep(1:nFreq)=1` is even clearer.